### DISCUSSION TITLE
More support for markdown, e.g., for live preview

### DISCUSSION THREAD

**Discussion by bagustris** [James-Yu/LaTeX-Workshop#4470] [Q&A]
Nowadays, It is very convenient to write Latex expressions in markdown files (even though some journals like Joss only provide templates in markdown format). Given that Github, via their gfm -- github flavored markdown, [supports latex via Mathjax](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions), along with pandoc, jekyll and others, I think having more support of markdown will be beneficial.

One of my current uses of Latex workshop for markdown files is to get code highlight by changing language mode from `markdown` to `markown_latex_combined`. However, this changing prevents me to have live markdown preview as I have in `markdown` language mode.

It will be very helpful if  we can directly preview the markdown files when using `markdown_latex_combined` although this language mode currently [**is not intended** to be used for .md files using LaTeX code](https://github.com/James-Yu/latex-workshop/wiki/ExtraFeatures#mixing-markdown-and-latex-code).

**Comment by James-Yu** [comment]
That should be a markdown extension issue. We don’t deal with markdown files.
